### ISSUE TITLE
Multiple GPU's enabled, but slow training

### ISSUE THREAD

**Issue by djSharma7** [ThilinaRajapakse/simpletransformers#244]
Hello,
I've been trying to train my model lately using GPU (p2.x8Large instance AWS) for the task.
And when i check the top usage of the machine, it shows only one cpu is in use, (rest 31 are idle)
My training set size (25k) 
Number of labels (12)
.
I have also enabled 

> use_cuda = True

>     n_gpu = 8

Is there anything I'm missing! 
` model = MultiLabelClassificationModel(
        "roberta",
        'roberta-base',
        # "outputs",
        num_labels=12,
        use_cuda=True,
        args={"reprocess_input_data": True,
              "overwrite_output_dir": True,
               'learning_rate': 3e-5,
              'num_train_epochs': 6,
              "use_cached_eval_features":False,
              'max_seq_length': 512,
              'fp16': False,
              'evaluate_during_training': True}
    )`

**Comment by ThilinaRajapakse** [comment]
You need to check GPU usage, not CPU.
